custom_components/tuya_local/devices/diivoo_dwv010.yaml

```diff
@@ -91,3 +115,47 @@ secondary_entities:
             value: "6 days"
           - dps_val: "7"
             value: "7 days"
+  - entity: number
+    name: Single use time 1
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 15
+        type: integer
+        name: value
+        unit: min
+        optional: true
+        range:
+          min: 0
+          max: 1440
+  - entity: number
+    name: Single use time 2
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 127
+        type: integer
+        name: value
+        unit: min
+        optional: true
+        range:
+          min: 0
+          max: 1440
+  - entity: sensor
+    class: time
+    name: Remaining irrigation time 2
```

Comment:
Time remaining is the standard wording used for such sensors.
The class should be `duration`